buildpack.md

```diff
@@ -251,21 +251,23 @@ The lifecycle MUST treat a layer with unset `types` as a `launch = false`, `buil
 The following table illustrates the behavior depending on the value of each flag.
 Note that the lifecycle only restores layers from the cache, never from the previous image.
 
-`build`   | `cache`  | `launch` | Metadata and SBOM** Restored | Layer Restored
-----------|----------|----------|------------------------------|---------------------
-true      | true     | true     | Yes - from the app image     | Yes* - from the cache
-true      | true     | false    | Yes - from the cache         | Yes - from the cache
-true      | false    | true     | No                           | No
-true      | false    | false    | No                           | No
-false     | true     | true     | Yes - from the app image     | Yes* - from the cache
-false     | true     | false    | Yes - from the cache         | Yes - from the cache
-false     | false    | true     | Yes - from the app image     | No
-false     | false    | false    | No                           | No
-
-\* The metadata and layer are restored only if the layer SHA recorded in the previous image matches the layer SHA recorded in the cache.
+| `build` | `cache` | `launch` | Metadata and SBOM** Restored | Layer Restored        |
+| ------- | ------- | -------- | ---------------------------- | --------------------- |
+| true    | true    | true     | Yes - from the app image***  | Yes* - from the cache |
+| true    | true    | false    | Yes - from the cache         | Yes - from the cache  |
+| true    | false   | true     | No                           | No                    |
+| true    | false   | false    | No                           | No                    |
+| false   | true    | true     | Yes - from the app image***  | Yes* - from the cache |
+| false   | true    | false    | Yes - from the cache         | Yes - from the cache  |
+| false   | false   | true     | Yes - from the app image     | No                    |
+| false   | false   | false    | No                           | No                    |
+
+\* If the previous image was found, the metadata and layer are restored only if the layer SHA recorded in the image matches the layer SHA recorded in the cache.
```

Comment:
```suggestion
```

The current level of detail is too much, but especially for the buildpack spec (buildpack authors don't need to care about this)